File: Q11/Q11.py
# Convert Celsius to Fahrenheit and vice versa.

def celsius_to_fahrenheit(celsius):
    return (celsius * 9/5) + 32

# Function to convert Fahrenheit to Celsius
def fahrenheit_to_celsius(fahrenheit):
    return (fahrenheit - 32) * 5/9

# Example usage
celsius = 25
fahrenheit = 77

print(f"{celsius}°C is {celsius_to_fahrenheit(celsius)}°F")
print(f"{fahrenheit}°F is {fahrenheit_to_celsius(fahrenheit)}°C")

File: Q11/Q11.ts
// Convert Celsius to Fahrenheit and vice versa.

function celsiusToFahrenheitTS(celsius: number): number {
  return (celsius * 9) / 5 + 32;
}

function fahrenheitToCelsiusTS(fahrenheit: number): number {
  return ((fahrenheit - 32) * 5) / 9;
}

const celsiusTS = 25;
const fahrenheitTS = 77;

console.log(`${celsiusTS}°C is ${celsiusToFahrenheitTS(celsiusTS)}°F`);
console.log(`${fahrenheitTS}°F is ${fahrenheitToCelsiusTS(fahrenheitTS)}°C`);
